src/views/dissolved-search/index.html

```diff
@@ -88,11 +88,13 @@ <h1 class="govuk-heading-xl">
               }
             ]
           }) }}
+          <script>readValue();</script>
       {{ govukButton({
         text: "Search",
         attributes: {
           "data-event-id": "dissolved-search-button",
-          id: "dissolved-search-button"
+          id: "dissolved-search-button",
+          onclick: "createItem(this)"
```

Comment:
I don't think that passing in `this` is necessary as the value is read using `document.getElementById`?